packages/expo-splash-screen/ios/EXSplashScreen/EXSplashScreenService.m

```diff
@@ -31,10 +32,17 @@ - (instancetype)init
 {
   if (self = [super init]) {
     _splashScreenControllers = [NSMapTable weakToStrongObjectsMapTable];
+    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(handleHideSplashScreen) name:kHideSplashScreenEvent object:nil];
```

Comment:
You've created this mechanism, but you're not using anywhere in this PR 👀 🤔

Reply:
forgot to revert! thanks for catching